src/components/joinus/JoinUsHero.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import HeroPlaceholder from "@/assets/joinus/hero_placeholder.jpg";
+
+type StaticImageData = {
+  src: string;
+};
+
+export const JoinUsHero: React.FC = () => {
+  return (
+    <div className="flex flex-col md:flex-row items-center justify-between p-8 md:p-12 space-y-8 md:space-y-0 md:space-x-8">
+      <div className="flex flex-col items-center py-8 md:px-4 text-center md:w-6/12">
+        <h2 className="text-5xl font-bold text-primary">Join Our Team!</h2>
+        <a
+          href="https://www.amazon.com"
+          target="_blank"
+          rel="noopener noreferrer"
+          className="mt-8 mb-8 md:mb-0 text-2xl"
+        >
+          Get involved with UBC BIOMOD →
+        </a>
+      </div>
+      <div className="flex items-center justify-center w-full md:w-[515px] md:h-[322px] h-64">
+        <img
+          src={(HeroPlaceholder as StaticImageData).src}
+          alt="UBC BIOMOD"
+          className="w-[515px] h-[322px] object-cover rounded-3xl"
```

Comment:
Don't do fixed pixel widths T_T This image isn't the best example since it's naturally square - the real photo will most likely be a landscape aspect ratio so you could get away with having `h-auto` and `w-full` + `md:w-1/2` or `md:w-1/3`. For this image specifically you can still do the same for width but fix the height to like 20rem or something.